bin/test.sh

```diff
@@ -7,19 +7,22 @@ cd "$(git rev-parse --show-toplevel 2>/dev/null)"
 
 if [ '--dev' = "$1" ]; then
     MAPNIK_DIR=$(dirname "$(python -c 'import mapnik; print(mapnik.__file__)')")
-     
+
     if [ ! -d venv ]; then
         virtualenv venv
     fi
     source venv/bin/activate
-     
+
     if [ ! -d venv/lib/python2.7/site-packages/mapnik ]; then
         ln -s "$MAPNIK_DIR" venv/lib/python2.7/site-packages/
     fi
 
      pip install --upgrade --requirement dev-requirements.txt
      PYTHONPATH=. py.test -v carto_renderer
 else
-    find . -name '*.pyc' -delete    
```

Comment:
Is cleanup removed or rendered unnecessary?

Reply:
Unnecessary.  We don't generate a file with the extension `.pyc` in the build process currently.